chef_master/source/platform_overview.rst

```diff
@@ -115,18 +115,10 @@ Compliance
 
 Chef Automate creates customizable reports that identify compliance issues, security risks, and outdated software. You can write your own compliance rules in InSpec, or you can get started quickly by using built-in profiles, which are predefined rule sets for a variety of security frameworks, such as Center for Internet Security (CIS) benchmarks, included as part of Chef Automate.
 
-Compliance reporting is currently done through two mechanisms: The integrated compliance reporting in Chef Automate 0.8.5 or later, and the standalone Chef Compliance server.
+Compliance reporting is currently done through the integrated compliance reporting in Chef Automate 0.8.5.
```

Comment:
Do we still want to include the version number here?

Reply:
Nope. Removed them from everywhere except for "Upgrade" and the release notes"